app.js

```diff
@@ -34,36 +34,40 @@ options:
   --verbose                    Print verbose output.
   --version                    Print the current version of Scandium, then exit.
   --output=<path>              Output built zip. Example: "--output scandium.zip"
+  --bucket=<name>              Use S3 bucket for deployment, needed for deployments over 50MB.
 `
 
 async function main () {
   const args = neodoc.run(usage, { laxPlacement: true })
   const listrOpts = (isCI || args['--verbose']) ? { renderer: listrVerboseRenderer } : {}
+  const usingS3 = Boolean(args['--bucket'])
 
   const createList = new Listr([
     tasks.parseOptions,
     tasks.packageApp,
     tasks.saveApp,
+    usingS3 && tasks.uploadToS3,
```

Comment:
Not needed since the `enable` flag is set on that task?

Reply:
Yes but the task will still be listed, so you can't see what will happen next, now with it filtered out you will only get the task that will be included

Reply:
Hmm, I thought that was what enabled was? 🤔

Isn't there two things? one for skipping, one for not showing at all?

Reply:
As I'm reading the documentation, it should behave that way: https://github.com/SamVerschueren/listr#enabling-tasks

> However, at first only the Yarn task will be visible.

Reply:
Nope, it will still show, and then skip it